pkg/api/handlers/types.go

```diff
@@ -354,21 +354,24 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 
 func LibpodToContainer(l *libpod.Container, infoData []define.InfoData) (*Container, error) {
 	imageId, imageName := l.Image()
-	sizeRW, err := l.RWSize()
-	if err != nil {
-		return nil, err
-	}
 
-	SizeRootFs, err := l.RootFsSize()
-	if err != nil {
-		return nil, err
-	}
+	var (
+		sizeRW     int64
+		SizeRootFs int64
+		err        error
+	)
 
 	state, err := l.State()
 	if err != nil {
 		return nil, err
 	}
 
+	// FIXME Why do the *Size() calls cause a panic I cannot catch on other states?
+	if state == define.ContainerStateRunning {
```

Comment:
This is a bug in the API, we should not work around it - we should find and fix the issue in Libpod.

Reply:
@jwhonce how about you and i take a closer look tomorrow